src/main/java/com/systemmeltdown/robotlib/triggers/DriverSlowTrigger.java

```diff
@@ -1,21 +1,26 @@
-package common;
+package com.systemmeltdown.robotlib.triggers;
 
 import edu.wpi.first.wpilibj.XboxController;
 import edu.wpi.first.wpilibj.GenericHID.Hand;
 import edu.wpi.first.wpilibj.buttons.Trigger;
-import common.RobotMap;
 
 public class DriverSlowTrigger extends Trigger {
   private XboxController controller;
+  private double triggerThreshold;
 
   public DriverSlowTrigger(XboxController controller) {
+    this(controller, 0.25);
```

Comment:
Let's take out this constructor. I don't think we should encourage people to use a hard-coded default.